test/run

```diff
@@ -9,6 +9,8 @@ if [ -n "$TEST_SCENARIO" ]; then
     test/check-cli TestLiveIso
   elif [ "$TEST_SCENARIO" == "qcow2" ]; then
     test/check-cli TestQcow2
+  elif [ "$TEST_SCENARIO" == "tar" ]; then
+    test/check-cli TestTar
```

Comment:
Add `rhel-7-7/tar` as manual in testmap.py and kick off this scenario manually to see if it works

Reply:
Does this mean only adding `rhel-7-7/tar` into `testmap.py` in cockpit/bots project? I'm not really sure what determines the manual/automatic execution in this case.

Reply:
When you open the testmaps.py file you will understand. There is a manual section and there are sections that get triggered automatically.

Reply:
Use `_manual` key in the dict contained in this file. Search for it to see examples.

Reply:
Thank you for the hint. I hope I understood that correctly: https://github.com/cockpit-project/bots/pull/86